preprocessing/tabular_data/feature_selection_tabular_data.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>Thus far, we have discussed two common approaches for obtaining lists of features ranked based on the feature importance scores. From these lists, we can infer unimportant features and eliminate them from our training, validation, and test sets.</p><p> </p><p> </p><p><strong>Can you elaborate this sentence a bit more?:</strong></p><p>We mention that both approaches constrain the removal of features to the dataset itself which is independant of the problem domain.</p><p> </p><p><strong>And there's a typo:</strong></p><p>independant --&gt; independent</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Done! I added a short example to explain this. </p>